### PR TITLE
Add a Twitter Api according to #364 - need testing

### PR DESCRIPTION
**Pull request by eamanu**
The `twitter_api.Twitter.PostTwitter(message)` receive a message and make a Tweet
Is necessary have a `twitter_api.secre`t like `github_pat.secret` with the next format
```
consumer_key='1234...'
consumer_secret='1234...'
access_token='12345...'
access_secret='1234...'
```

Todo List:
1. Test twitter_api
1. create a Twitter Apps (https://apps.twitter.com/) for ChaosBot and complete the twitter_api.secret

### CODE REVIEW

**Comment by rudehn** [comment]
I would recommend putting your twitter keys into a json file. Then loading them is as simple as `keys = json.load(file_pointer)`

**Comment by phil-r** [comment]
To add token/secret we can use `encryption` module added in #227
Documentation is here: https://github.com/chaosbot/Chaos/pull/321/files (never was merged 😢)

**Comment by hongaar** [comment]
@phil-r would be nice if we could add this to the wiki!

**Comment by phil-r** [comment]
@hongaar added: https://github.com/chaosbot/Chaos/wiki/Encryption

**Review by syvb — APPROVED** [review]

**Comment by bobjoe27326** [comment]
:+1:

**Comment by eamanu** [comment]
@phil-r I don't undestand how can I use the encryptation with the Twitter's keys. Can you make a little sample?

**Review by syvb — APPROVED** [review]

**Comment by Swizz** [comment]
Is Chaosbot :-1: referenced as a negative vote ? Oo

File: chaos.py [hunk 1]
```diff
@@ -69,12 +74,17 @@ def main():
 
     api = gh.API(settings.GITHUB_USER, settings.GITHUB_SECRET)
 
+    # Api Twitter
+    api_twitter = ta.API_TWITTER(settings.TWITTER_API_KEYS_FILE)
+
     log.info("checking if I crashed before...")
+    ta.Twitter.PostTwitter("checking if I crashed before...", api_twitter.GetApi())
```
Comment (andrewda) [inline chaos.py:81]: Do we really want it to be this verbose? I don't think it needs to tweet this out every time it starts up.

File: chaos.py [hunk 2]
```diff
@@ -69,12 +74,17 @@ def main():
...
 
     # check if chaosbot is not on the tip of the master branch
     check_for_prev_crash(api, log)
 
     log.info("starting up and entering event loop")
+    ta.Twitter.PostTwitter("starting up and entering event loop", api_twitter.GetApi())
```
Comment (andrewda) [inline chaos.py:87]: Same as above.

**Review by andrewda — CHANGES_REQUESTED** [review]
Maybe a tweet when a PR gets merged, or created. Maybe also change the current tweets to "Starting up..." instead of two simultaneous tweets when starting up.

**Comment by rudehn** [comment]
I think anything posted to twitter should be important. The info posted right now is better suited to log files

**Comment by eamanu** [comment]
@rudehn @andrewda I will make that Chaosbot make tweets when there are  PR or Merges. What are the files that make the PR or files of interest to change?

File: chaos.py (hunk 1 above)
Comment (eamanu) [reply chaos.py:81]: I think like @rudehn. Anything posted to twitter should be important. A post in twitter isn't a cost or some extra.

File: chaos.py (hunk 2 above)
Comment (eamanu) [reply chaos.py:87]: Same as above :+1:

**Comment by PlasmaPower** [comment]
@eamanu It probably should tweet when any PR gets merged. Alternatively, we could ignore PRs with only documentation changes (only `*.md` files touched).

**Review by eamanu — COMMENTED** [review]
I think like @rudehn. Anything posted to twitter should be important. A post in twitter isn't a cost or some extra

**Comment by mark-i-m** [comment]
I suspect that posting crashes might be good...

**Comment by dbpokorny** [comment]
> I suspect that posting crashes might be good...

https://www.youtube.com/watch?v=bQlpDiXPZHQ

Life Call Commercial "I've fallen and I can't get up!"

> I'm having interpreter pain!

> I've halted, and I can't restart!

**Pull request closed without merging**